cli/ballerina-cli/src/main/resources/cli-help/ballerina-tool-update.help

```diff
@@ -0,0 +1,15 @@
+NAME
+       bal-tool-update - Update to the latest patch version within the minor
```

Comment:
Bal tool update must update to the latest version compatible with the current distribution.

Reply:
Changed